cypress/tests/upstream/vm-adv-search.cy.ts

```diff
@@ -0,0 +1,227 @@
+import { K8S_KIND, OS_IMAGES_NS, TEST_NS } from '../../utils/const/index';
+
+const SEARCH_NS = 'auto-search-ns';
+const SAVE_NAME = 'Saved search CNV QE';
+const SAVE_DESCR = 'CNV QE Testing save';
+const SRCH_VMS = ['centos-stream9-sssss', 'centos-stream10-sssss', 'fedora-sssss'];
```

Comment:
create for each name a const then you can use it for all the tests below

Reply:
done